include/libtorrent/piece_picker.hpp

```diff
@@ -97,9 +97,9 @@ namespace libtorrent {
 			// priority factor
 			prio_factor = 3,
 			// max blocks per piece
-			// there are counters in downloading_piece that only have 15 bits to
+			// there are counters in downloading_piece that only have 16 bits to
 			// count blocks per piece, that's restricting this
-			max_blocks_per_piece = (1 << 15) - 1
```

Comment:
the reason this wasn't caught early was because this `15` should have been `14`. The bitfield was probably changed without this changing.